homeassistant/components/comelit/coordinator.py

```diff
@@ -95,9 +95,17 @@ async def _async_update_data(self) -> T:
             await self.api.login()
             return await self._async_update_system_data()
         except (CannotConnect, CannotRetrieveData) as err:
-            raise UpdateFailed(repr(err)) from err
+            raise UpdateFailed(
+                translation_domain=DOMAIN,
+                translation_key="update_failed",
+                translation_placeholders={"error": repr(err)},
```

Comment:
what would an error be?

Reply:
Seems so obvious to me, that I may miss something.
`CannotConnect` can underline different reasons, as well as `CannotRetrieveData`.